modules/oxide/src/less/skins/content/default/content.less

```diff
@@ -18,27 +18,6 @@ table:not([cellpadding]) td {
   padding: .4rem;
 }
 
-/* Set default table styles if a table has a positive border attribute
-   and no inline css */
-table[border]:not([border="0"]):not([style*="border-width"]) th,
-table[border]:not([border="0"]):not([style*="border-width"]) td {
-  border-width: 1px;
-}
-
-/* Set default table styles if a table has a positive border attribute
-   and no inline css */
-table[border]:not([border="0"]):not([style*="border-style"]) th,
-table[border]:not([border="0"]):not([style*="border-style"]) td {
-  border-style: solid;
-}
-
-/* Set default table styles if a table has a positive border attribute
-   and no inline css */
-table[border]:not([border="0"]):not([style*="border-color"]) th,
-table[border]:not([border="0"]):not([style*="border-color"]) td {
-  border-color: #ccc;
```

Comment:
A nestačilo by to přepsat na `black` tady? Nejsem si jistá, jestli když se nastaví `table_default_styles` v tom editor configu, jestli to pak bude i v preview.

Reply:
no kdybych to přepsala na black, tak černé tabulky se solid ohraničením budou v pohodě - ale pokud máš třeba tabulku s růžovou border color, tak se tam tyhle styly stejně vecpou, pokud je to trasparent.

líbilo se mi, že by se tam hodili default styly, které si nastavíš v settings editoru a ne který je definován na random tady, což řeší ten druhý PR pomocí nasetování stylů table elementu.

preview by mělo být v pohodě - manuálně jsem to zkoušela simulovat v dev tools a myslím, že ta oprava v druhém PR, by se měla postarat o to (už vlastně při prvním průchodu parserem).

takže za mě to pojďme zkusit :)